Automattic/jetmon
# Stats: Add support to output stats into `statsd`

This PR adds logging into StatsD, so we can monitor the health of Jetmon better.

This PR also adds StatsD image to the local docker instance, so we can monitor the stats locally if we need to.

First version includes basic stats that we'll extend once we make sure this approach will work for our objectives.

### Testing instructions:

* Apply PR
* `docker compose build`
* `docker compose up`
* Check `https://localhost:8088` for the default Graphite Browser UI. 
* Make sure you can see some metrics being logged in `stats -> com -> jetpack -> jetmon -> docker -> jetmon -> dev -> com` in the Tree View on the side.
* Set the interval to the last 10 minutes to see anything in the UI. (It's not the best :( )
